.github/workflows/validate.yaml

```diff
@@ -2,16 +2,40 @@ name: Validate
 
 on:
   push:
-  pull_request:
-  schedule:
-    - cron: "0 0 * * *"
```

Comment:
I don't think that this is really useful to run the tests everyday.
But if you prefer to, I can revert that change.

Reply:
I originally just copied the workflow from the HACS docs. My guess would be that this is meant to ensure that the app developer gets notified if HACS introduces breaking changes to the repo validation. Now, I don't know if or when HACS re-validates repos after the initial inclusion to their default repo list and what happens to repos if the validation fails, but I think a regular check for the HACS validation is still useful.

However, you are right that this doesn't make sense for the tests. I think the best way would be to just separate the validate and test workflows, and remove the cron schedule from the test workflow.

Reply:
@NiklasReiche Yeah, that would make sense! I think I will do something similar; but maybe run it just once every week.

However, since I don't set any version in `requirements.txt` (I can't control what people will use in their AppDaemon config), I think it would be better to let them in the same workflow along with the tests.

Si if there is in the futur a breaking change on that side too, we would get notified.

Reply:
Good point, I hadn't thought of that! Once a week seems like a good tradeoff for the tests, so I'll do the same. Thanks!